src/runtime/contrib/cblas/cblas.cc

```diff
@@ -31,6 +31,9 @@ extern "C" {
 #else
 #include <cblas.h>
 #endif
+#if USE_MKL_DNN == 1
```

Comment:
I think code logic here need a small change as well:
```
#if USE_MKL_BLAS == 1
#include <mkl_cblas.h>
#else if USE_MKL_DNN == 1
#include <dnnl.h>
#else
#include <cblas.h>
#endif
```

Reply:
@ZhennanQin As @icemelon9 mentioned [here](https://github.com/apache/incubator-tvm/pull/4323#discussion_r345946543), we need both cblas and dnnl because the latter is used for sgemm only.